pkg/prometheus/resource_selector.go

```diff
@@ -659,6 +659,18 @@ func validateProberURL(url string) error {
 	return nil
 }
 
+func validateServer(server string) error {
+	parsedURL, err := url.Parse(server)
+	if err != nil {
+		return fmt.Errorf("can not parse server: %s", err.Error())
```

Comment:
I think its enough to just return the error since it should say unable to parse url
```suggestion
		return err
```